infra/bracin/src/bracin/core.clj

```diff
@@ -1,7 +1,13 @@
 (ns bracin.core
+  (:require [ring.adapter.jetty :as jetty])
   (:gen-class))
 
+(defn handler
```

Comment:
Not very important as this will evolve but out of curiosity, why is this no simply

```
(defn hander [req]
  {:status 200, :body "Hello, world."})
```
?

Reply:
Yes, this would work too. Actually right now the simplest form is likely to not have a separate handler and just inline the lambda.

The reason I like having the current form is that this is where we introduce configuration. Moving forward the `-main` method would for example get the Google Auth client secret from the environment and pass it `handler` (which I suppose should be called `make-handler`).

I'll just inline the lambda for now.